test/functional/rawtransactions.py

```diff
@@ -64,6 +64,20 @@ def run_test(self):
         # This will raise an exception since there are missing inputs
         assert_raises_jsonrpc(-25, "Missing inputs", self.nodes[2].sendrawtransaction, rawtx['hex'])
 
+        ############################
+        # getrawtx with block hash #
+        ############################
+
+        # make a tx by sending then generate 2 blocks; block1 has the tx in it,
+        # presumably
+        tx = self.nodes[2].sendtoaddress(self.nodes[1].getnewaddress(), 1)
+        [ block1, block2 ] = self.nodes[2].generate(2)
+        self.sync_all()
+        # We should be able to get the raw transaction by providing the correct block
+        assert self.nodes[0].getrawtransaction(tx, True, block1)
```

Comment:
nit: I think it's better to assert on the actual value here (ie verify that the getrawtransaction returned the correct transaction rather than returned anything). The following should do that:

`assert_equal(self.nodes[0].getrawtransaction(tx, True, block1)['txid'], tx)`